component/prometheus/exporter/gcp/gcp.go

```diff
@@ -0,0 +1,77 @@
+package gcp
+
+import (
+	"time"
+
+	"github.com/grafana/agent/component"
+	"github.com/grafana/agent/component/prometheus/exporter"
+	"github.com/grafana/agent/pkg/integrations"
+	"github.com/grafana/agent/pkg/integrations/gcp_exporter"
+)
+
+func init() {
+	component.Register(component.Registration{
+		Name:    "prometheus.exporter.gcp",
+		Args:    Arguments{},
+		Exports: exporter.Exports{},
+		Build:   exporter.New(createExporter, "gcp"),
+	})
+}
+
+func createExporter(opts component.Options, args component.Arguments) (integrations.Integration, error) {
+	a := args.(Arguments)
+	return a.Convert().NewIntegration(opts.Logger)
+}
+
+type Arguments struct {
+	// Google Cloud project ID from where we want to scrape metrics from
```

Comment:
nit: we can remove these comments, I don't thi they are very useful. If you want to keep them, at least should start with the name of the argument (e.g `ProjectIDs are Google Cloud project IDs from where we want to scrape metrics from`)

Reply:
I will remove the comments from the static mode GCP exporter config as well to keep things consistent.